src/arch/mod.rs

```diff
@@ -17,14 +19,18 @@ use crate::data::{
 };
 use Type::*;
 
-// NOTE: this is required by the standard to always be one
+/// Size of a `char` in bytes;
+///
+/// Note: The standard requires this to always be one.
 const CHAR_SIZE: u16 = 1;
 
 // TODO: allow this to be configured at runtime
+/// The target triple for the configured target.
```

Comment:
I don't think this is very helpful. Maybe link to an explanation of target triple instead: https://clang.llvm.org/docs/CrossCompilation.html#target-triple or https://docs.rs/target-lexicon/0.10.0/target_lexicon/struct.Triple.html

Also, this is the host target, not any configured target.

Reply:
My new explanation for target triple may be a little long winded.

Reply:
I like it :)